docs/awesome/tools.md

```diff
@@ -9,36 +9,67 @@
 另有第三方的 UUID 生成器 [uuidgenerator.net](https://www.uuidgenerator.net)
 
 ## 第三方图形客户端
+<Tool
+    url="https://github.com/2dust/v2rayN"
+    name="V2rayN"
+    :platforms="['win']"
+    description="【正在更新】V2RayN 是一个基于 V2Ray和Singbox等一系列内核的 Windows 客户端。"
+/>
 
 <Tool
     url="https://github.com/Qv2ray/Qv2ray"
     name="Qv2ray"
     :platforms="['linux','win','mac']"
-    description="跨平台 V2Ray 客户端，支持 Linux、Windows、macOS，可通过插件系统支持 SSR / Trojan / Trojan-Go / NaiveProxy 等协议"
-/>
-
-<Tool
-    url="https://github.com/SagerNet/SagerNet"
-    name="SagerNet"
-    :platforms="['android']"
-    description="SagerNet 是一个基于 V2Ray 的 Android 通用代理应用。"
+    description="【正在更新】跨平台 V2Ray 客户端，支持 Linux、Windows、macOS，可通过插件系统支持 SSR / Trojan / Trojan-Go / NaiveProxy 等协议（可能停止更新）"
 />
 
 <Tool
-    url="https://github.com/2dust/v2rayN"
-    name="V2rayN"
-    :platforms="['win']"
-    description="V2RayN 是一个基于 V2Ray 内核的 Windows 客户端。"
+    url="https://github.com/MatsuriDayo/nekoray"
+    name="nekoray"
+    :platforms="['linux','win','mac']"
+    description="【正在更新】支持SOCKS (4/4a/5),HTTP(S),Shadowsocks,VMess,VLESS,Trojan,TUIC,NaïveProxy ( Custom Core ),Hysteria2 ( Custom Core or sing-box ),Custom Outbound,Custom Config,Custom Core,v2ray Core"
 />
 
 <Tool
     url="https://github.com/v2rayA/v2rayA"
     name="v2rayA"
     :platforms="['linux','win','mac']"
-    description="基于 web GUI 的跨平台 V2Ray 客户端，在 Linux 上支持全局透明代理，其他平台上支持系统代理。"
+    description="【正在更新】基于 web GUI 的跨平台 V2Ray 客户端，在 Linux 上支持全局透明代理，其他平台上支持系统代理。"
+/>
+<Tool
+    url="https://github.com/yanue/V2rayU/tree/master"
+    name="v2rayU"
+    :platforms="['mac']"
+    description="【正在更新】MacOS平台的v2ray客户端 支持vmess,vless,ss,ssr协议,支持socks5协议,支持xlts"
+/>
+
+<Tool
+    url="https://github.com/2dust/v2flyNG"
+    name="v2flyNG"
+    :platforms="['android']"
+    description="【正在更新】v2flyNG 是一个基于 V2Ray 的 Android 通用代理应用。"
+/>
+
+<Tool
+    url="https://github.com/2dust/v2rayNG"
+    name="v2rayNG"
+    :platforms="['android']"
+    description="【正在更新】v2rayNG 是一个基于 Xray 的 Android 通用代理应用。"
```

Comment:
I was unable to verify this tool is open source. Please make sure this tool is open source and remove duplication.

Reply:
I am not sure if v2flyNG is open source (it seems to just be v2rayNG with a different core), but v2rayNG is open source

Reply:
Already removed v2flyNG because I couldn't determine if it's open-source, and added https://github.com/dyhkwong/Exclave to the tool list.